src/coreclr/ilasm/asmparse.y

```diff
@@ -47,8 +47,8 @@
 %token <string>  DOTTEDNAME     /* System.Object */
 %token <binstr>  QSTRING        /* "Hello World\n" */
 %token <string>  SQSTRING       /* 'Hello World\n' */
-%token <int32>   INT32          /* 3425 0x34FA  0352  */
-%token <int64>   INT64          /* 342534523534534      0x34FA434644554 */
+%token <int32>   INT32_V        /* 3425 0x34FA  0352  */
+%token <int64>   INT64_V        /* 342534523534534      0x34FA434644554 */
```

Comment:
I would instead recommend `INT32_T` and `INT64_T` for naming.

Reply:
I assume _T is token here. I use _V for value. but whatether, let maintainers decide.

Reply:
My 2 cents are: I personally like `_V` for value. If it were `_T` I would expect a token representing the type, not a value.

Reply:
"_V" is fine.